atom/browser/atom_browser_context.cc

```diff
@@ -156,6 +157,12 @@ content::BrowserPluginGuestManager* AtomBrowserContext::GetGuestManager() {
   return guest_manager_.get();
 }
 
+net::SSLConfigService* AtomBrowserContext::CreateSSLConfigService() {
+  if (!ssl_config_service_)
+    ssl_config_service_.reset(new AtomSSLConfigService);
+  return ssl_config_service_.get();
```

Comment:
I think the ownership of `ssl_config_service_` should be transferred to `URLRequestContextGetter`?